src/plugins/dashboard/public/dashboard_top_nav/internal_dashboard_top_nav.tsx

```diff
@@ -313,10 +321,51 @@ export function InternalDashboardTopNav({
       });
     }
     if (showWriteControls && managed) {
-      allBadges.push(getManagedContentBadge(dashboardManagedBadge.getTooltip()));
+      const badgeProps = {
+        ...getManagedContentBadge(dashboardManagedBadge.getAriaLabel(), false),
+        onClick: () => setIsPopoverOpen(!isPopoverOpen),
+        onClickAriaLabel: dashboardManagedBadge.getAriaLabel(),
+        iconOnClick: () => setIsPopoverOpen(!isPopoverOpen),
+        iconOnClickAriaLabel: dashboardManagedBadge.getAriaLabel(),
+      } as TopNavMenuBadgeProps;
+      allBadges.push({
+        renderCustomBadge: ({ badgeText }) => {
+          const badgeButton = <EuiBadge {...badgeProps}>{badgeText}</EuiBadge>;
+          return (
+            <EuiPopover
+              button={badgeButton}
+              isOpen={isPopoverOpen}
+              closePopover={() => setIsPopoverOpen(false)}
+              panelStyle={{ maxWidth: 350 }}
+            >
+              <EuiText size="s">{dashboardManagedBadge.getText()}</EuiText>
+              <EuiText size="s">
+                <EuiLink
+                  aria-label={dashboardManagedBadge.getDuplicateText()}
+                  onClick={() => {
+                    dashboard.runInteractiveSave(viewMode);
+                  }}
+                >
+                  {dashboardManagedBadge.getDuplicate()}
+                </EuiLink>
+                {dashboardManagedBadge.getDuplicateText()}
+              </EuiText>
```

Comment:
Now that the link is inline like it is, I think we should use the `FormattedMessage` component with an `EuiLink` to get this done in a way that works nicely for `i18n` 🙇 In other languages, the "Duplicate" link **may not** belong at the beginning of the sentence - we should never separately translate two parts of a sentence and assume that slapping them together like this will work after being translated.

Reply:
++ @Heenawter. We don't need to repeat `this dashboard` in the second sentence. With the link on `Duplicate` only, this would look like:

---

Elastic manages this dashboard.
[Duplicate](url) it to make changes.